release-plz.toml

```diff
@@ -0,0 +1,28 @@
+[workspace]
+# disable the changelog for all packages, will only enable for shpool
+changelog_update = false
+
+[changelog]
+header = ""
+body = """
+{{ package }} ({{ version | trim_start_matches(pat="v") }}) unstable; urgency=low
+{% for group, commits in commits | group_by(attribute="group") %}
+  {{ group | upper_first }}
+{% for commit in commits -%}
+{%- if commit.scope %}
+  * *({{commit.scope}})* {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message }}{%- if commit.links %} ({% for link in commit.links %}[{{link.text}}]({{link.href}}) {% endfor -%}){% endif %}
+{%- else %}
+  * {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message }}
+{%- endif %}
+{%- endfor %}
+{% endfor %}
+ -- Ethan Pailes <pailes@google.com>  {{ now() | date(format="%a, %d %b %Y %T %z") }}
```

Comment:
@ethanpailes I put your name here to keep it the same as the existing change logs. Let me know if this is good or you want something else.

Reply:
Dig into debian manual for a bit. Multiple authors works by prefixing each set of changes with an author. I don't think it's worth the hassle.

"Shpool Authors" is good but is there an email alias we could use?